cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -145,16 +145,25 @@ spec:
 #      deviceFilter: "^sd."
   # Override Ceph configs. These can be overridden by the user at runtime using the Ceph CLI, and
   # Rook will set them back to the values set here when the operator restarts.
+  # The user is advised to set 'osd_pool_default_pg_num' and 'osd_pool_default_pgp_num' based on
+  # the Ceph docs and their anticipated cluster configuration. Bare minimum defaults are set below.
+  # https://docs.ceph.com/docs/master/rados/operations/placement-groups/#a-preselection-of-pg-num
   configOverrides:
     - who: global
-      option: debug ms
-      value: "1/5"
-    - who: osd.0
-      option: debug_osd
-      value: "10"
-    - who: mon.a
-      option: debug-mon
-      value: "10"
+      option: osd_pool_default_pg_num
```

Comment:
I thought we agreed on keeping Ceph's default, thus not exposing `osd_pool_default_pg_num` nor `osd_pool_default_pgp_num`. Did I miss something?

Reply:
This is part of the doc updates to indicate to users that they should pay attention to these  configs. We could comment them out if that seems better.

Reply:
@leseb @travisn I would like some thoughts about this.